src/olympia/devhub/forms.py

```diff
@@ -317,7 +317,6 @@ class Meta:
 
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
```

Comment:
you can delete the `__init__` method now - it's not doing anything beyond the superclass.

Reply:
sure thing